Makefile

```diff
@@ -180,6 +180,10 @@ lint:
 		-A clippy::question-mark \
 		-A clippy::uninlined-format-args
 
+# Lints the code using Clippy and automatically fix some simple compiler warnings.
+lint-fix:
+	EXTRA_CLIPPY_OPTS="--fix --allow-staged --allow-dirty" $(MAKE) lint
```

Comment:
I can't find a reference for a built-in variable called `MAKE`, but it seems to work on Linux. I presume you also tested on macOS?

Reply:
That should be sufficient, as Linux uses GNU make, and macOS uses BSD make.

Reply:
Yep, works on MacOS.